### DISCUSSION TITLE
blog/querying-azure-digital-twins-with-powerbi/

### DISCUSSION THREAD

**Discussion by giscus** [Sealjay/sealjay-blog#14] [Comments]
# blog/querying-azure-digital-twins-with-powerbi/

A Digital Twin is a digital replica of a physical object, space, or abstract process. It allows us to model the state of these things virtually, and then respond to events as they happen in real time

https://sealjay.com/blog/querying-azure-digital-twins-with-powerbi/

**Comment by Jan99Jan** [comment]
I am currently trying to connect my Digital Twin Explorer to Power BI. I don't quite understand how to carry out steps 3-5. Can you explain the steps in more detail?
Thank you very much!
